cms/templates/js/xblock-access-editor.underscore

```diff
@@ -0,0 +1,5 @@
+<div class="incontext-editor-action-wrapper">
+    <a href="" class="unit-container visibility-button" title="<%- gettext('Access Settings') %>">
```

Comment:
It would be much better to use a button instead of an a-link. An empty href is an a11y violation, and this is really acting as a button.

This issue is pervasive in Studio. If changing this to a button causes styling issues that cannot be easily overcome, leave it as an a-link, and it can be changed when all the rest are changed. But when you attempt to change to a button, you will probably find it easiest to also change the edit button beside it to a button (since they share CSS, and it has the same problem).